MAINTAINERS.md

```diff
@@ -1,3 +1,21 @@
+# Initial setup
+
+```
+git clone git@github.com:endojs/endo.git
+cd endo
+yarn
+```
+
+Yarn workspaces are used. Running yarn in the root will install and hoist dependencies in the ./node_modules and not in packages themselves.
```

Comment:
```suggestion
Endo is a yarn workspaces repository. Running yarn in the root will install and hoist most dependencies up to the root `node_modules` and instead of the packages themselves.
```